scripts/idl-analyze.py

```diff
@@ -272,6 +286,9 @@ def handle_interface(analysis, iface):
         print('Syntax error in IDL', fname, file=sys.stderr)
         raise e
         sys.exit(1)
+    print('XPIDL: Parsed', fname, 'into', len(r.productions), 'productions', file=sys.stderr)
     for p in r.productions:
```

Comment:
Did you want to keep these?

Reply:
Yeah, the lack of output from the `idl-analyze.py` step was a hindrance in setting this up and has also been a hindrance in previous variations of XPIDL (I think due to merge-analyses making the non-identical files per-platform) and IPDL failures (which is an ongoing problem of rust/python parser inconsistencies, although a different analyzer entirely).  Not a major hindrance, but being able to grep/search for a file of interest in the index log and get (or informatively not get!) a hit that explains what happened in the pipeline I hope will make analyzing failures even easier for people who might ramp up in the future on searchfox.

Reply:
Whoops, forgot the example I wanted to paste.  Here's what `make build-tests-repo` provides:

```
+ cat /home/vagrant/index/tests/idl-files      
+ parallel /vagrant/scripts/idl-analyze.py /home/vagrant/index/tests '/home/vagrant/index/tests/files/{}' '>' '/home/vagrant/index/tests/analysis/{}'                                         
XPIDL: No C++ analysis data found for /home/vagrant/index/tests/files/xpidl/nsISupports.idl                                                                                                   
XPIDL: No C++ analysis data found for /home/vagrant/index/tests/files/xpidl/nsrootidl.idl                                                                                                     
XPIDL: Parsed /home/vagrant/index/tests/files/xpidl/xpctest_params.idl into 5 productions                                                                                                     
+ echo 0 
```